backend/src/repository/analysis_collection.py

```diff
@@ -303,23 +299,37 @@ def attach_third_party_link(self, analysis_name: str, third_party_enum: str, lin
         if not analysis:
             raise ValueError(f"Analysis with name {analysis_name} does not exist")
 
-        if third_party_enum == "MONDAY_COM":
-            third_party_enum = ThirdPartyLinkType.MONDAY_COM
-        elif third_party_enum == "PHENOTIPS_COM":
-            third_party_enum = ThirdPartyLinkType.PHENOTIPS_COM
+        # if third_party_enum == "MONDAY_COM":
```

Comment:
This can get deleted now that you migrated the enumeration to be passed in and not the string.

Reply:
Thanks for catching this. I meant to delete it after testing, but it slipped by.